app/licenses.yml

```diff
@@ -143,3 +143,7 @@
   copyrightHolder: #COPYRIGHT_HOLDER#
   license: #LICENSE#
   skip: true
+- artifact: com.tomoima.debot:+
```

Comment:
CI detected error `Execution failed for task ':app:checkLicenses'.`

I think the following is correct.

`com.tomoima.debot:+` => `com.tomoima.debot:+:+`